client/src/components/UserProfile.jsx

```diff
@@ -0,0 +1,187 @@
+import TextInput from './TextInput';
+import styled from 'styled-components';
+import { useEffect, useState } from 'react';
+import profilePic from '../images/ProfilePic.jpg';
+import Button from './Button';
+import DateInput from './DateInput';
+import dayjs from 'dayjs';
+const URL = process.env.REACT_APP_URL;
+const USER_ID = 9999;
+
+const UserProfile = () => {
+  const [climber, setClimber] = useState({
+    climber_id: USER_ID,
+    user_name: '',
+    first_name: '',
+    last_name: '',
+    birthday: '',
+    boulder_start_date: '',
+  });
+
+  const [errorSave, setErrorSave] = useState(false);
+  const [errorGet, setErrorGet] = useState(false);
+  const [success, setSuccess] = useState(false);
+
+  const fetchClimber = () => {
+    fetch(`${URL}/climber/${USER_ID}`)
+      .then(res => res.json())
+      .then(data => {
+        data.birthday = dayjs(data.birthday).format('YYYY-MM-DD');
+        data.boulder_start_date = dayjs(data.boulder_start_date).format(
+          'YYYY-MM-DD'
+        );
+        setClimber(data);
+      })
+      .catch(error => {
+        setErrorGet(true);
+      });
+  };
+
+  useEffect(() => {
+    fetchClimber();
+    setSuccess(false);
+    setErrorGet(false);
+    setErrorSave(false);
+  }, []);
+
+  const saveClimberToDatabase = () => {
+    fetch(`${URL}/climber/${USER_ID}/`, {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify(climber),
+    })
+      .then(response => response.json())
+      .then(data => {
+        setClimber(data);
+        setSuccess(true);
+      })
+      .catch(error => {
+        setErrorSave(true);
+      });
+  };
+
+  return (
+    <>
```

Comment:
I think you don't need the fragments. Please check :)

Reply:
You're right!